### ISSUE TITLE
is.opera

### ISSUE THREAD

**Issue by mato75** [arasatasaygin/is.js#145]
returns false, when used with opera browser.

**Comment by FelisPhasma** [comment]
Try something like this:
```
navigator.userAgent.indexOf("Opera");

// OR:

if (window.opera && opera.toString() == "[object Opera]"){
  // ...
}
```

**Comment by planttheidea** [comment]
I don't think @arasatasaygin is maintaining this project anymore, nor has there been any activity on the account since May. I have forked the project and have rebranded it as isit.js ... you'd be surprised how many "is" variants exist on NPM. I have updated the checker for Opera based on the [MDN documentation](https://developer.mozilla.org/en-US/docs/Browser_detection_using_the_user_agent).

Feel free to try it out if you'd like, just npm install initjs. I have also rearchitected it to be more ES2015 module based for ease of maintenance, so if you may find bugs, but I'll be on top of them.

**Comment by stgogm** [comment]
This works fine for newer versions of Opera:

```js
navigator.userAgent.indexOf(' OPR/') >= 0
```

**Comment by jdalton** [comment]
Fixed by #197.
